code/game/machinery/bank_machine.dm

```diff
@@ -105,8 +105,11 @@
 
 	switch(action)
 		if("siphon")
-			say("Siphon of station credits has begun!")
-			start_siphon(ui.user)
+			if(is_station_level(src.z) || is_centcom_level(src.z))
```

Comment:
For consistency, do this for the "halt" action too.

Reply:
I'll go with what melbert said, sounds like a better idea